algtest/util.c

```diff
@@ -30,3 +30,17 @@ FILE *open_csv(const char *filename, const char *header)
     fprintf(file, "%s\n", header);
     return file;
 }
+
+FILE *open_bin(const char *filename)
+{
+    char path[256];
+    strncpy(path, options.outdir, strlen(options.outdir) + 1);
```

Comment:
This smells like possible overflow...  better PATH_MAX? (and perhaps snprintf for the whole size). Just saying :)